osmosis-1/osmosis-1.assetlist.json

```diff
@@ -1551,6 +1551,39 @@
         "OSMO": 725
       }
     },
+    {
+      "description": "The native EVM, governance and staking token of the Acrechain",
+      "denom_units": [
+        {
+          "denom": "ibc/BB936517F7E5D77A63E0ADB05217A6608B0C4CF8FBA7EA2F4BAE4107A7238F06",
+          "exponent": 0,
+          "aliases": ["aacre"]
+        },
+        {
+          "denom": "acre",
+          "exponent": 18
+        }
+      ],
+      "base": "ibc/BB936517F7E5D77A63E0ADB05217A6608B0C4CF8FBA7EA2F4BAE4107A7238F06",
+      "name": "Acre",
+      "display": "acre",
+      "symbol": "ACRE",
+      "ibc": {
+        "source_channel": "channel-0",
+        "dst_channel": "channel-490",
+        "source_denom": "aacre"
+      },
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/acrechain/images/acre.svg",
+	      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/acrechain/images/acre.png"
+      },
+      "coingecko_id": "arable-protocol",
+      "keywords": [
+        "osmosis-main",
+        "osmosis-frontier"
```

Comment:
osmosis-frontier keyword requires that a pool to be specified. Please remove this keyword or specify a pool.

Reply:
We will create a pool and add pool ID soon.

Reply:
Pool created and commited.
https://github.com/osmosis-labs/assetlists/pull/289/commits/44e818777ecb2138dfd649ba12d9f7ff13b966ea
We were not able to add more liquidity since fronteir was not working to add liquidity.